rules/mandatory-object-comma.js

```diff
@@ -0,0 +1,117 @@
+                    };
+
+                    findStringBetweenBrackets(stringAfterBracketsPair, stringsBetweenBrackets);
+                };
+
+                findStringBetweenBrackets(string, stringsBetweenBrackets);
+
+                return stringsBetweenBrackets;
+            }
+
+            const changeTwigVariablesToUnicode = (content, variables) => {
+                let modifiedContent = content;
+
+                if (Array.isArray(variables)) {
+                    variables.forEach(variable => {
+                        const changedStringSymbolLength = 2;
+
+                        modifiedContent = modifiedContent.replace(
+                            variable, `&#123;&#123;${variable.slice(
+                                changedStringSymbolLength,(-1)*changedStringSymbolLength)}&#125;&#125;`);
+                    })
+                }
+
+                return modifiedContent;
+            };
+
+            const createFirstLevelObjectData = data => {
+                const firstLevelObjects = [];
+                let partOfStringObject = '';
+
+                data.forEach( str => {
+                    partOfStringObject = partOfStringObject ? partOfStringObject + str.slice(1) : str;
+
+                    const openingCurlyBrackets = partOfStringObject.match(/\{/g);
+                    const openingSquareBrackets = partOfStringObject.match(/\[/g);
+                    const closingCurlyBrackets = partOfStringObject.match(/\}/g);
+                    const closingSquareBrackets = partOfStringObject.match(/\]/g);
+                    const countOfOpeningCurlyBracket = openingCurlyBrackets ? openingCurlyBrackets.length : 0;
+                    const countOfOpeningSquareBracket = openingSquareBrackets ? openingSquareBrackets.length : 0;
+                    const countOfClosingCurlyBracket = closingCurlyBrackets ? closingCurlyBrackets.length : 0;
+                    const countOfClosingSquareBracket = closingSquareBrackets ? closingSquareBrackets.length : 0;
+
+                    if (countOfOpeningCurlyBracket !== countOfClosingCurlyBracket ||
+                        countOfOpeningSquareBracket !==  countOfClosingSquareBracket) {
+                        return;
+                    }
+
+                    firstLevelObjects.push(partOfStringObject);
+                    partOfStringObject = '';
+                });
+
+                return firstLevelObjects;
+            };
+
+            const checkCommaAfterLastObjectsParameter = objectStrings => {
+                const commaAfterLastParameterReg = /[a-zA-Z0-9\]\}]{1}[' ']{0,}[\r]{0,1}[' ']{0,}[\n]{1,}[' ']{0,}[\]|\}]{1,}/g;
+
+                objectStrings.forEach(objectString => {
+                    if (!commaAfterLastParameterReg.test(objectString)) {
+                        return;
+                    }
+
+                    objectString = objectString.replace(/&#123;/g, '{');
+                    objectString = objectString.replace(/&#125;/g, '}');
```

Comment:
Please use `objectString = objectString.replace(/&#123;/g, '{').replace(/&#125;/g, '}');` instead of it.